packages/react/src/context/languageContext.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { PropsWithChildren, createContext, useContext, useEffect, useRef, useState } from 'react';
+import React, { PropsWithChildren, createContext, useContext, useEffect, useState } from 'react';
```

Comment:
I thought I mentioned this somewhere but I think it's best if this languageContext is moved inside header folder because this only relates to Header usage and isn't an actual general-use languageContext that we export for the users. I think the HDS naming for context files is also to start with capital letter, so this should be `LanguageContext.tsx`.